app/models/miq_provision_virt_workflow.rb

```diff
@@ -348,7 +348,7 @@ def allowed_templates(options = {})
     rails_logger('allowed_templates', 1)
     log_allowed_template_list(allowed_templates_list)
 
-    MiqPreloader.preload(allowed_templates_list, [:operating_system])
+    MiqPreloader.preload(allowed_templates_list, [:hardware, :operating_system])
```

Comment:
Note, we need cpu/mem/disk information for the extra columns defined on line 303 above:

```
VM_OR_TEMPLATE_EXTRA_COLS = %i[mem_cpu cpu_total_cores v_total_snapshots allocated_disk_storage].freeze
```